Framework/Kernel/test/TypedValidatorTest.h

```diff
@@ -43,7 +43,7 @@ class TypedValidatorTest : public CxxTest::TestSuite {
   void test_shared_ptr_is_passed_successfully_to_concrete_validator() {
     Mantid::Kernel::IValidator_sptr valueChecker =
         std::make_shared<SharedPtrTypedValidator>();
-    const std::shared_ptr<Holder> testPtr = std::make_shared<Holder>();
+    const std::shared_ptr<Holder> &testPtr = std::make_shared<Holder>();
```

Comment:
I think we don't want to reference capture here for the same reason as the use of `make_shared` above.